helm/alloy/templates/secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.alloy.alloy.secret.create }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "alloy.fullname" . }}
+  labels:
+    {{- include "alloy.labels" . | nindent 4 }}
+    app.kubernetes.io/component: config
+data:
+  {{- if .Values.alloy.alloy.secret.content }}
+  {{- .Values.alloy.alloy.secret.content | nindent 2 }}
+  {{- else }}
+  {{- .Files.Get "config/example.secret" | nindent 2 }}
```

Comment:
I would prefer that you require the `.Values.alloy.alloy.secret.content` to not be empty than provide an incorrect config though

Reply:
That's a good point so I did move the secret example into `helm/alloy/examples/mimir-rules`